src/screen.ts

```diff
@@ -37,15 +44,22 @@ const logTestingPlaygroundURL = (element = getDocument().body) => {
   )
 }
 
-const initialValue = {debug, logTestingPlaygroundURL}
+const initialValue: {
+  [key in keyof typeof queries | 'debug' | 'logTestingPlaygroundURL']?: Function
+} = {debug, logTestingPlaygroundURL}
+
 export const screen =
-  typeof document !== 'undefined' && document.body
+  typeof document !== 'undefined' && document.body // eslint-disable-line @typescript-eslint/no-unnecessary-condition
     ? getQueriesForElement(document.body, queries, initialValue)
-    : Object.keys(queries).reduce((helpers, key) => {
+    : typedKeysOf(queries).reduce<typeof initialValue>((helpers, key) => {
```

Comment:
There's a reason why `Object.keys` returns `string[]` and not an array of literals: https://stackoverflow.com/questions/55012174/why-doesnt-object-keys-return-a-keyof-type-in-typescript. Let's stick with Object.keys and annotate the types properly (casting, `@ts-expect-error` etc).

Reply:
I think in this case perhaps a `helpers[key as keyof typeof initialValue]` might work best. A @ts-expect-error might work too, however, anyone familiar with the Object.keys caveat may be instantly familiar with the nature and reason of the inline cast. 

Though I'm down for whatever! I'll push my proposed change and you can let me if it works.

In addition, doing the cast here makes the explicit typing of `initialValue` above and the explicit type param passed to `reduce` here unneccessary as well.

Reply:
Double post question: Do y'all care about squashing commits? EG should I squash the aforementioned upcoming commit into the one last pushed?

Reply:
> Double post question: Do y'all care about squashing commits? EG should I squash the aforementioned upcoming commit into the one last pushed?

During review on github it's usually best to not rebase so that notifications only display the most recent changes. We squash on merge anyway.

Reply:
> I think in this case perhaps a helpers[key as keyof typeof initialValue] might work best.

Yeah. `@ts-expect-error` would also hide potential future errors. But let's document why we think the cast is safe and link the `Object.keys` caveat.

Reply:
Pushed